javascript/house-style.md

```diff
@@ -195,6 +195,46 @@ We don't do this:
 const uniqueIDs = new Set([1, 2, 1, 2]);
 ```
 
+#### JS Doc
+
+Although your JavaScript code should be self-documenting, it is highly
```

Comment:
"it is highly recommended" -> "we highly recommend that you" 

(House style, "[Prefer the active voice to the passive voice](https://github.com/springernature/frontend-playbook/blob/main/writing/house-style.md#plain-english)")